src/imagery/i.eodag/i.eodag.py

```diff
@@ -615,60 +585,60 @@ def parse_query(query=None):
     :rtype: Dict[str, List(Tuple(str, str))]
 
     """
-    VALID_OPERATORS = ["eq", "ne", "ge", "gt", "le", "lt"]
-    DEFAULT_OPERATOR = "eq"
+    valid_operators = ["eq", "ne", "ge", "gt", "le", "lt"]
+    default_operator = "eq"
     query_list = []
     if query is None:
         return query_list
-    for parameter in map(str.strip, options["query"].split(",")):
-        if parameter == "":
+    for parameter in map(str.strip, query.split(",")):
+        if not parameter:
             continue
         try:
             key, values = map(str.strip, parameter.split("="))
-        except Exception as e:
+        except ValueError as e:
             gs.debug(e)
             gs.fatal(_("Queryable <{}> could not be parsed").format(parameter))
         if key == "start":
             try:
                 start_date = normalize_time(values)
-                query_list.append(("start", (start_date, DEFAULT_OPERATOR)))
-            except Exception as e:
+                query_list.append(("start", (start_date, default_operator)))
+            except ValueError as e:
                 gs.debug(e)
                 gs.fatal(
                     _(
-                        "Queryable <{}> could not be parsed\nDate must be ISO formated"
-                    ).format(parameter)
+                        "Queryable <{}> could not be parsed\nDate must be ISO formated",
+                    ).format(parameter),
                 )
             continue
         if key == "end":
             try:
                 end_date = normalize_time(values)
-                query_list.append(("end", (end_date, DEFAULT_OPERATOR)))
-            except Exception as e:
+                query_list.append(("end", (end_date, default_operator)))
+            except ValueError as e:
                 gs.debug(e)
                 gs.fatal(
                     _(
-                        "Queryable <{}> could not be parsed\nDate must be ISO formated"
-                    ).format(parameter)
+                        "Queryable <{}> could not be parsed\nDate must be ISO formated",
+                    ).format(parameter),
                 )
             continue
         operator = None
         values_operators = []
         for value in map(str.strip, values.split("|")):
```

Comment:
```suggestion
        values_operators = []
        for value in map(str.strip, values.split("|")):
            operator = None
```
Must have slipped from before, default value for an operator should be None, later translated to 'eq'.